src/persist/src/client.rs

```diff
@@ -35,34 +35,37 @@ use crate::storage::SeqNo;
 /// all clients have been dropped, whichever comes first.
 ///
 /// NB: The methods below are executed concurrently. For a some call X to be
-/// guaranteed as "previous" to another call Y, X's response must have been
-/// received before Y's call started.
-#[derive(Clone)]
+/// guaranteed as "previous" to another call Y, X's future must have been
```

Comment:
nit/random thought: we can probably express the "previous" relation in terms of `SeqNos` for mutating commands right?

Reply:
IMO this is more actionable for a user of persist. SeqNos describe what happened, but this is talking about how you use persist to get the ordering guarantees you want